imqsauth/http.go

```diff
@@ -614,11 +630,67 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 		} else {
 			central.Central.Log.Errorf("Could not set group roles for %v: %v", groupname, err)
 			authaus.HttpSendTxt(w, http.StatusNotAcceptable, fmt.Sprintf("Could not set group roles for %v: %v", groupname, err))
+			return
 		}
 	} else {
 		central.Central.Log.Warnf("Group '%v' not found: %v", groupname, e)
 		authaus.HttpSendTxt(w, http.StatusNotFound, fmt.Sprintf("Group '%v' not found: %v", groupname, e))
+		return
 	}
+
+	broadCastGroupChange(central, r, groupname)
+}
+
+func broadCastGroupChange(central *ImqsCentral, r *httpRequest, groupname string) {
+
+	identities, err := central.Central.GetAuthenticatorIdentities()
+	if err != nil {
+		central.Central.Log.Errorf("Unable to broadcast change, unable to read identities: %v", err)
+		return
+	}
+
+	// now find all identities which belong to the group in question
```

Comment:
more nitpicky: I find that when reading code, using 'now' to introduce a comment often interrupts comprehension. I believe the only case where one should use 'now', or 'next', etc, is when there is a trail of comments that forms a narrative, and where it's important to understand that whole thread of comments. Usually when I read "now..." I can't help but think that it was a comment placed there before the code was written. As in.. "now I need to write some code to do X". One usually doesn't write post-hoc comments in that manner. I warned you it was nitpicky!